Dockerfile

```diff
@@ -7,12 +7,23 @@ ENV PYTHONUNBUFFERED 1
 
 # Install non python dependencies
 RUN apt-get update
-RUN apt-get install -y libpq-dev gcc curl
+RUN apt-get install -y libpq-dev gcc curl gpg
 
 # Fetch Amazon RDS certificate chain
 RUN curl https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem -o /usr/local/share/amazon-certs.pem
 RUN chmod a=r /usr/local/share/amazon-certs.pem
 
+# Install MSSQL ODBC 18 Driver
+RUN mkdir -m 0755 -p /etc/apt/keyrings/
+RUN curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg
+RUN echo "deb [signed-by=/etc/apt/keyrings/microsoft.gpg] https://packages.microsoft.com/debian/12/prod bookworm main" | tee /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get update
+RUN ACCEPT_EULA=Y apt-get install -y msodbcsql18
+
+# modify openssl config to allow TLSv1 connection for TransitMaster server
+RUN sed -i 's/\[openssl_init\]/# [openssl_init]/' /etc/ssl/openssl.cnf
+RUN echo '\n\n[openssl_init]\nssl_conf = ssl_sect\n\n[ssl_sect]\nsystem_default = ssl_default_sect\n\n[ssl_default_sect]\nMinProtocol = TLSv1\nCipherString = DEFAULT@SECLEVEL=0\n' >> /etc/ssl/openssl.cnf
+
```

Comment:
There are probably other ways to update this openssl config, but I found this one to be the most straightforward....

Reply:
i looked around for other ways to modify these conf files, and it seems like using `echo` and `sed` is a fine way for doing simple changes like this.